src/test/java/htsjdk/samtools/cram/build/ContainerFactoryTest.java

```diff
@@ -23,45 +24,45 @@
         final int mappedSequenceId = 0;  // arbitrary
         final ReferenceContext mappedRefContext = new ReferenceContext(mappedSequenceId);
         final int mappedAlignmentStart = 1;
-        final int mappedAlignmentSpan = CramCompressionRecordUtil.READ_LENGTH_FOR_TEST_RECORDS + TEST_RECORD_COUNT - 1;
+        final int mappedAlignmentSpan = READ_LENGTH_FOR_TEST_RECORDS + TEST_RECORD_COUNT - 1;
```

Comment:
This could use a comment explaining how it's derived. Its only when you look at the implementation of the `create` test utilities that it makes sense. Or better yet can we eliminate or make the dependency more explicit somehow ? Anyway, a comment would be fine if not.